### PR TITLE
Hindi translation for the website

### PR DESCRIPTION
**Pull request by toshitapandey**
## Description 

- Hindi translation for the website

### More Details

- Update locale.rb to include Hindi as a language
- Add devise.hi.yml, kaminari.hi.yml, hi.yml, devise.inevitable.hi.yml for Hindi translations
- Add Hindi as language to all the language files 

### Corresponding Issue

https://github.com/ifmeorg/ifme/issues/830

## Test Coverage

✅ 
🚫

### CODE REVIEW

**Comment by julianguyen** [comment]
This is so awesome! Thank you Toshita ^^

**Comment by julianguyen** [comment]
Can you put this PR in draft mode? :)

**Comment by julianguyen** [comment]
Looks like there's a YAML linting error actually:
```
locales/hi.yml
  697:83    error    no new line character at the end of file  (new-line-at-end-of-file)
```

**Comment by julianguyen** [comment]
Any updates? :)

**Comment by toshitapandey** [comment]
@julianguyen Yeah on it. Will be done by EOD today.

**Comment by julianguyen** [comment]
By the way, there's no rush to get everything done right away! I just wanted to check in and see where things were at!

File: config/locales/devise.hi.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    passwords:
+      no_token: 'आप पासवर्ड रीसेट ईमेल से आए बिना इस पृष्ठ तक नहीं पहुंच सकते। यदि आप एक पासवर्ड रीसेट ईमेल से आते हैं, तो कृपया सुनिश्चित करें कि आपने पूर्ण URL प्रदान किया है।'
+      send_instructions: 'आपको कुछ मिनटों में अपना पासवर्ड रीसेट करने के निर्देशों के साथ एक ईमेल प्राप्त होगा।'
+      send_paranoid_instructions: 'यदि आपका ईमेल पता हमारे डेटाबेस में मौजूद है, तो आपको कुछ ही मिनटों में आपके ईमेल पते पर एक पासवर्ड रिकवरी लिंक प्राप्त होगा।'
+      updated: 'आपका पासवर्ड सफलतापूर्वक बदल दिया गया है। अब आप साइन इन हैं।'
+      updated_not_active: 'आपका पासवर्ड सफलतापूर्वक बदल दिया गया है।'
+    registrations:
+      destroyed: 'अलविदा! आपका खाता सफलतापूर्वक रद्द कर दिया गया है। हम जल्द ही फिर से मिलने की उम्मीद करते है।'
+      signed_up: 'स्वागत है! आपने सफलतापूर्वक साइन अप कर लिया है।'
+      signed_up_but_inactive: 'आपने सफलतापूर्वक साइन अप कर लिया है। हालाँकि, हम आपको साइन नहीं कर सकते क्योंकि आपका खाता अभी तक सक्रिय नहीं है।'
```
Comment (amanjeev) [inline config/locales/devise.hi.yml]: > हालाँकि, हम आपको साइन नहीं
should this be ?
> ...हम आपको साइन इन नहीं

File: config/locales/devise.hi.yml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        subject: 'पासवर्ड रीसेट करने के निर्देश'
+      unlock_instructions:
+        subject: 'अनलॉक करने के निर्देश'
+      password_change:
+        subject: 'पासवर्ड बदला गया'
+    omniauth_callbacks:
+      failure: 'आपको %{kind} से प्रमाणित नहीं किया जा सकता क्योंकि %{reason}।'
+      success: '%{reason} खाते से सफलतापूर्वक प्रमाणित किया गया।'
+    passwords:
+      no_token: 'आप पासवर्ड रीसेट ईमेल से आए बिना इस पृष्ठ तक नहीं पहुंच सकते। यदि आप एक पासवर्ड रीसेट ईमेल से आते हैं, तो कृपया सुनिश्चित करें कि आपने पूर्ण URL प्रदान किया है।'
```
Comment (amanjeev) [inline config/locales/devise.hi.yml]: I see other places use "लिंक" so maybe "URL" can be changed to "लिंक" here?

File: config/locales/devise.hi.yml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+      destroyed: 'अलविदा! आपका खाता सफलतापूर्वक रद्द कर दिया गया है। हम जल्द ही फिर से मिलने की उम्मीद करते है।'
+      signed_up: 'स्वागत है! आपने सफलतापूर्वक साइन अप कर लिया है।'
+      signed_up_but_inactive: 'आपने सफलतापूर्वक साइन अप कर लिया है। हालाँकि, हम आपको साइन नहीं कर सकते क्योंकि आपका खाता अभी तक सक्रिय नहीं है।'
+      signed_up_but_locked: 'आपने सफलतापूर्वक साइन अप कर लिया है। हालाँकि, हम आपको साइन नहीं कर सके क्योंकि आपका खाता बंद है।'
+      signed_up_but_unconfirmed: 'आपके ईमेल पते पर एक पुष्टिकरण लिंक के साथ एक संदेश भेजा गया है। कृपया अपने खाते को सक्रिय करने के लिए लिंक का अनुसरण करें।'
+      update_needs_confirmation: 'आपने अपना खाता सफलतापूर्वक अपडेट कर दिया है, लेकिन हमें आपके नए ईमेल पते को सत्यापित करना होगा। कृपया अपने ईमेल की जाँच करें और अपने नए ईमेल पते की पुष्टि करने के लिए पुष्टि लिंक का पालन करें।'
+      updated: 'आपका खाता सफलतापूर्वक अपडेट कर दिया गया है।'
+    sessions:
+      signed_in: 'सफलतापूर्वक हस्ताक्षर किए गए।'
+      signed_out: 'सफलतापूर्वक हस्ताक्षर किए गए।'
```
Comment (amanjeev) [inline config/locales/devise.hi.yml]: Maybe just "साइन आउट" here

File: config/locales/devise.hi.yml (hunk 2 above)
Comment (toshitapandey) [reply config/locales/devise.hi.yml]: You're right. I'd change this.

File: config/locales/devise.hi.yml (hunk 1 above)
Comment (toshitapandey) [reply config/locales/devise.hi.yml]: Oh, I missed it.

File: config/locales/devise.hi.yml (hunk 3 above)
Comment (toshitapandey) [reply config/locales/devise.hi.yml]: Maybe we should do same for signed in too. Suggestions?

Comment (amanjeev) [reply config/locales/devise.hi.yml]: > Maybe we should do same for signed in too. 
I agree. I believe most will get confused by "हस्ताक्षर".

**Comment by julianguyen** [comment]
Feel free to merge whenever you think it's ready to go :D
Thanks @amanjeev for your reviews!

**Comment by toshitapandey** [comment]
> Feel free to merge whenever you think it's ready to go :D

@julianguyen Let's sit on it for a day or two more, and if someone else finds an issue we'll fix it and I'll also go through it a few more times, just in case.

**Pull request merged**